### DISCUSSION TITLE
Cannot see the thumbnail and the description of my custom panel in the panel list page

### DISCUSSION THREAD

**Discussion by killerJek** [foxglove/community#835] [Q&A]
Hi,

I'm not able to visualize the panel thumbnail neither the panel description in the panel search, the description is visible only in the `Setting/Extensions`. I saw that the image has been built and is in the correct local folder.
Thanks for the help. 
![image](https://github.com/foxglove/community/assets/102426527/6dff6db2-d926-416f-9850-85b5675e4f64)

**Comment by jtbandes** [comment]
We don't currently support custom thumbnails or descriptions for extension panels.

**Reply by defunctzombie** [reply]
@jtbandes did we make a feature request issue for this internally?

**Reply by jtbandes** [reply]
https://linear.app/foxglove/issue/FG-5849
